src/python/pants/backend/python/goals/setup_py.py

```diff
@@ -308,6 +310,45 @@ class RunSetupPyResult:
     output: Digest  # The state of the chroot after running setup.py.
 
 
+@enum.unique
+class FirstPartyDependencyVersionScheme(enum.Enum):
+    EXACT = "exact"  # i.e., ==
+    COMPATIBLE = "compatible"  # i.e., ~=
+    ANY = "any"  # i.e., no specifier
+
+
+class PythonDistributionSubsystem(Subsystem):
+    """Options for packaging wheels/sdists from a `python_distribution` target."""
+
+    options_scope = "python-distribution"
```

Comment:
Should this be `python_distribution` to match the target type name? 

Related: I want to make a change that normalizes dashes in scope names to underscores, so that they behave similarly to the option name itself. (i.e., `--sco-pe-na-me` and `[sco_pe].na_me` both work, as well as, `[sco-pe].na_me` for backwards compat).

Reply:
I had it as `python_distribution` before, but changed it because every single scope I've seen using `-`, so it seemed confusing to be inconsistent.

That flexibility would be nice. We will want to decide the canonical format used in documentation.

I don't think this is worth changing to `python_distribution` until we add the new flexibility you describe.